tasks/update-cr-status/tests/test-update-cr-status-nonexistent-status-field.yaml

```diff
@@ -0,0 +1,67 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-update-cr-status-nonexistent-status-field
+  annotations:
+    test/assert-task-failure: "run-task"
+spec:
+  description: |
+    Run the update-cr-status task with to update a field that does not exist in the
```

Comment:
Typo: `with to update a field` -> `to update a field`?

Reply:
Fixed, thanks